mava/configs/env/connector.yaml

```diff
@@ -1,7 +1,7 @@
 # ---Environment Configs---
 defaults:
   - _self_
-  - scenario: con-10x10x5a # [con-5x5x3a, con-7x7x5a, con-10x10x5a, con-15x15x10a]
+  - scenario: con-5x5x3a # [con-5x5x3a, con-7x7x5a, con-10x10x10a, con-15x15x23a]
```

Comment:
I am worried that someone will set the time limit in this file and think it is what will be used however later time_limit is controlled by the param in the scenario.py

Reply:
Yes, this time limit is redundant.